lib/SparkPost/Transmission.php

```diff
@@ -63,12 +63,22 @@ private function __construct() {}
 	 */
 	private static function getHttpClient() {
 		if(!isset(self::$request)) {
-			self::$request = new Client();
+			$hostConfig = SparkPost::getConfig();
+			$defaultOptions = [
+				'base_uri'=>self::getBaseUrl($hostConfig),
+				'headers' =>[
+					'Authorization' => $hostConfig['key'],
+					'Content-Type' => 'application/json',
+					'User-Agent' => 'SparkPost PHP SDK v0.1.2'
```

Comment:
I think it's good to have UA in there. Should we align UAs across libraries? Regarding the version, this is an ``0.x.x`` pre-stable release, but should we just bump to ``0.2.0`` with this? Does everyone generally follow semver in this community like they do in Node.js?